Fix bogus expression:

CMake Error at libproxy/cmake/modules.cmk:26 (math):
  math cannot parse the expression: "0+0+0+": syntax error, unexpected end of
  file (6).

Index: libproxy/cmake/modules/pacrunner_duktape.cmk
--- libproxy/cmake/modules/pacrunner_duktape.cmk.orig
+++ libproxy/cmake/modules/pacrunner_duktape.cmk
@@ -17,5 +17,7 @@ else()
       link_directories(${DUKTAPE_LIBRARIES})
       link_libraries(duktape)
     endif()
+  else()
+    set(DUKTAPE_FOUND 0)
   endif()
 endif()
